<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node pkg="wheelctrl_4ws" type="wheelctrl_4ws" name="wheelctrl_4ws" output="screen">
    <param name="control_frequency" value="30"/>
    <param name="lost_time_threshold" value="500"/>
    <param name="body_width" value="0.440"/>
  </node>

</launch>
